docs/source/api_reference/models.rst

```diff
@@ -266,6 +266,12 @@ WGAN
 .. autoclass:: deepchem.models.WGAN
   :members:
 
+CNNModule
```

Comment:
CNNModule should be added to https://github.com/deepchem/deepchem/blob/master/docs/source/api_reference/layers.rst

Reply:
https://github.com/deepchem/deepchem/blob/master/docs/source/api_reference/layers.rst#torch-layers